src/changelog.ts

```diff
@@ -164,24 +164,57 @@ function getTagUrl(repoUrl: string, tag: string) {
  * @param repoUrl - The URL for the GitHub repository.
  * @param tagPrefix - The prefix used in tags before the version number.
  * @param releases - The releases to generate link definitions for.
+ * @param [versionBeforePackageRename] - A version string of the package before being renamed.
```

Comment:
Sorry, I think I missed this when the suggestion was made. I don't think we should have to provide brackets around arguments in TypeScript files. This is required when using JSDoc, but neither VSCode nor `typedoc` honors this syntax when interpreting JSDoc blocks, and we also have TypeScript types to indicate when something is optional. See comment here: https://github.com/MetaMask/auto-changelog/pull/159#pullrequestreview-1680555339. Thoughts on removing this (and other places where this has been added)?

Reply:
Reverted https://github.com/MetaMask/auto-changelog/commit/b51a4fa27dff4b46f3452e6b799a3d3c014c06bb as per discussion in https://github.com/MetaMask/auto-changelog/pull/159#pullrequestreview-1680555339.